recipe/meta.yaml

```
@@ -19,31 +19,40 @@ requirements:
   host:
     - python >=3.5
     - pip
+    - hatchling >=1.12.2
+    - hatch-vcs >=0.2.0
+    - wheel
   run:
     - python >=3.5
-    - six
-    - cryptography
+    - cryptography >=41
 
 test:
+  imports:
+    - conda_content_trust
   source_files:
     - tests
   requires:
     - pytest
     - pytest-cov
     - ruamel_yaml
```

```suggestion
    - securesystemslib
```

I don't think `ruamel_yaml` is used [anymore](https://github.com/search?q=repo%3Aconda%2Fconda-content-trust%20ruamel&type=code).

---

Also `pyproject.toml` [pins it](https://github.com/conda/conda-content-trust/blob/ebe40421dbf47a72314ea92d5fd29519652afb21/pyproject.toml#L51C34-L51C40).

---

can't add this yet, `securesystemslib=0.13.1` is not available on conda-forge ([only `0.28.0`](https://anaconda.org/conda-forge/securesystemslib/files)), we will work to relax this in the future

---

I guess we could just add it without the pin, it appears to work in the [CI](https://github.com/conda/conda-content-trust/blob/ebe40421dbf47a72314ea92d5fd29519652afb21/tests/requirements.txt#L7)

---

added in https://github.com/conda-forge/conda-content-trust-feedstock/pull/1/commits/037d9c2304e41e40844dbbabc92796ecef48c6b2